bin/status_job_submit.py

```diff
@@ -110,16 +110,28 @@ def compose_command(config, args,  hdfs_commands, logger=None):
     if args.Method == "insert":
         argo_mongo_client = ArgoMongoClient(args, config, logger, ["status_metrics", "status_endpoints", "status_services", "status_endpoint_groups"])
         argo_mongo_client.mongo_clean_status(mongo_uri)
-    
+
     # MongoDB method to be used when storing the results, either insert or upsert
     cmd_command.append("--mongo.method")
     cmd_command.append(args.Method)
-    
+
     # add the hdfs commands
     for command in hdfs_commands:
         cmd_command.append(command)
         cmd_command.append(hdfs_commands[command])
 
+    # ams proxy
```

Comment:
ams_interval and ams_batch config params should also be passed as cli arguments in status streaming call (as done in metric ingest and metric)

Reply:
fixed

Reply:
Looks great!